buildlib/pr/main.yml

```diff
@@ -121,8 +194,9 @@ stages:
     - template: tests.yml
       parameters:
         name: gpu
-        demands: ucx_gpu -equals yes
+        demands: ucx_gpu_test -equals yes
```

Comment:
should we remove this now? seems it runs on both docker and bare metal?

Reply:
yes, it need for test, I have one agent with one label `ucx_gpu_test` and this label always free

Reply:
still needed?